src/test/java/org/assertj/core/internal/int2darrays/Int2DArrays_assertNotEmpty_Test.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.internal.int2darrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.error.ShouldNotBeEmpty.shouldNotBeEmpty;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.internal.Int2DArrays;
+import org.assertj.core.internal.Int2DArraysBaseTest;
+import org.junit.jupiter.api.Test;
+
+
+/**
+ * Tests for <code>{@link Int2DArrays#assertNotEmpty(AssertionInfo, int[][])}</code>.
+ * 
+ * @author Alex Ruiz
+ * @author Joel Costigliola
+ * @author Maciej Wajcht
+ */
+public class Int2DArrays_assertNotEmpty_Test extends Int2DArraysBaseTest {
```

Comment:
similar comments as made in `Int2DArrays_assertDoesNotContain_at_Index_Test`

Reply:
done